src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsPluginComponent.kt

```diff
@@ -48,7 +52,10 @@ class SettingsPluginComponent {
     )
     private val telemetrySeparator = JXTitledSeparator("Telemetry")
     private var telemetryEnabledCheckbox = JCheckBox("Enable telemetry")
-    private val telemetryPathTextField = JTextField(TestGenieSettingsState.DefaultSettingsState.telemetryPath)
+    private val fileChooserDescriptor = FileChooserDescriptor(false, true, false, false, false, false)
+    private val textBrowseFolderListener = TextBrowseFolderListener(fileChooserDescriptor)
+    private val telemetryPathChooser = TextFieldWithBrowseButton()
+    private val telemetryPathLabel = JBLabel("Specify the folder path for telemetry data")
```

Comment:
Right now OK button just quits the settings, but makes the field to be the home directory. Maybe you can at least specify to the user that the default is the home directory, so that such quitting would not be so surprising?

Reply:
Right now if the person left a blank line, then the path will be restored to the previous path. If the path was changed to path A, then it will be restored to path A not to home directory. 
The user should be able to see that the default is home directory when they first turn on telemetry. 
I wrote why I do not want to have pop us messages in my pull request description. 

"When the telemetry path is empty, the reset happens without informing the user that the reset occurred, i.e. no popup message is shown. This is done in order not to disturb the user in case the empty path was just a mistake. If needed the message can be added."

Reply:
Alright